cats-effect/shared/src/main/scala/org/specs2/matcher/IOMatchers.scala

```diff
@@ -0,0 +1,52 @@
+package org.specs2
+package matcher
+
+import cats.effect.IO
+import cats.effect.unsafe.IORuntime
+import cats.syntax.all.*
+import org.specs2.concurrent.ExecutionEnv
+import org.specs2.execute.Failure
+import org.specs2.execute.Result
+import org.specs2.execute.Success
+import org.specs2.text.Regexes.matchesSafely
+
+import scala.concurrent.Future
+
+trait IOMatchers extends ValueChecks:
```

Comment:
It would be nice to add some comments on that trait to explain how it is different from the `IOTimedMatchers` trait.